dataFields/ABFieldFormulaCore.js

```diff
@@ -81,7 +81,14 @@ module.exports = class ABFieldFormulaCore extends ABField {
       delete values[this.columnName];
    }
 
-   format(rowData) {
+   /**
+    * @method format
+    * both calculate and format the data input based of user settings
+    * for this field.
+    * @param {obj} rowData a key=>value hash of the current values.
```

Comment:
In my comment headers, I've been trying to change our formatting to be like so:
```
* @param {type} {param}
*        {start the description here, but keep the lines to <= 80 cols }
*        {use as many lines as you need to }
```

It keeps to the ESLINT rule of <= 80 cols and actually gives us a bit more space for writing the comments.

Reply:
I made the change to the formatting.